src/detectors.py

```diff
@@ -87,3 +87,20 @@ def analyze(self, ast):
     if(NATVisitor.foundTransfer_to_NullAddress):
       return True
     return False
+
+class HiddenMintDetector(Detector): 
+  class HiddenMintVisitor:
+    def __init__(self):
+      self.foundConstant = False
+
+    def visitUserDefinedTypeName(self, node):
+      if(node.namePath == "_totalSupply.add"):
```

Comment:
Is `_totalSupply` a standard variable name for total supply? If not maybe we could change this to look for variables whose name contains "totalSupply" to be a bit more general. Also can we look for totalSupply being incremented by the binary operator addition as well to discover mints like `_totalSupply=_totalSupply+amount`

Reply:
made some changes to check for a binary addition on _totalSupply and modified the contract to include the functions I'm trying to detect.